resources/js/src/app/helper/debounce.js

```diff
@@ -0,0 +1,24 @@
+import {defaultValue, isNullOrUndefined} from "./utils";
+
+export function debounce(callback, timeout)
```

Comment:
Kannst du bei den Funktionen in den Helper Klassen vllt immer einen kleinen Satz in ein Doc packen damit man die schneller überfliegen kann wenn man mal was braucht?